boards/arm/nrf9160_pca20035/CMakeLists.txt

```diff
@@ -4,12 +4,8 @@
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 
-zephyr_library()
 
-zephyr_library_sources_ifdef(
-	CONFIG_TRUSTED_EXECUTION_NONSECURE
-	board_nonsecure.c)
-
-zephyr_library_sources_ifdef(
-        CONFIG_TRUSTED_EXECUTION_SECURE
-	board_secure.c)
+if(CONFIG_TRUSTED_EXECUTION_NONSECURE)
+  zephyr_library()
+  zephyr_library_sources(board_nonsecure.c)
```

Comment:
Could you quickly clarify why the board configuration does not apply to the secure firmware?

Reply:
It's because it sets up MAGPIO configurations that require BSD lib, which can only be used as non-secure.

Reply:
Can we use the BOARD name in this cmake statement, instead of the TrustZone Kconfig?

Reply:
Yes, fixed